packages/ui/src/stories/Carousel.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import { styled } from '@storybook/theming'
+import { get } from 'lodash'
+import { Carousel as Component } from '@open-condo/ui/src'
+import { colors } from '@open-condo/ui/src/colors'
+
+const AVAILABLE_COLORS = [
+    colors.green['3'],
+    colors.blue['3'],
+    colors.red['3'],
+    colors.teal['3'],
+    colors.cyan['3'],
+]
+
+const Slide = styled.h1<{ index: number }>`
+  background: ${(props => AVAILABLE_COLORS[props.index % AVAILABLE_COLORS.length])};
+  border-radius: 8px;
```

Comment:
This is a default border radius for all of the components. Maybe move this value to constant styles?

Reply:
Same as above I suppose